Makefile

```diff
@@ -27,10 +29,11 @@ deploy-twine-test: bdist_wheel deploy-requirements
 		--username ${TWINE_USERNAME} \
 		--password ${TWINE_PASSWORD}
 
-doc-requirements:
-	$(PYTHON) -m pip install -r build_tools/doc/doc_requirements.txt
+documentation: version
+	$(DOCKER) run -v $(HERE):/pmdarima --rm alkalineml/pmdarima-doc-base:latest /bin/bash -c "cd pmdarima && make install && make docker-documentation"
```

Comment:
If you set `-w /pmdarima` you can simplify the entrypoint to `/bin/bash -c "make install && make docker-documentation"`

Reply:
Nice! Implemented in 4a9f027